### PR TITLE
Enable webgl tests.

### PR DESCRIPTION
**Pull request by jdm**
…ling ones.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10806)

### CODE REVIEW

**Comment by jdm** [comment]
I'm going to run this through try a few times to make sure it doesn't hurt us.

**Comment by perlun** [comment]
@jdm - this one has gone unmergeable. So maybe this is the answer to your question "what should I work on" :wink:

**Comment by perlun** [comment]
(but ah, like you said, perhaps waiting to let other stuff get merged might be pretty neat anyway.)

File: tests/wpt/metadata/webgl/conformance-1.0.3/conformance/more/functions/bufferDataBadArgs.html.ini [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [bufferDataBadArgs.html]
   type: testharness
+  disabled: https://github.com/servo/servo/issues/10660
```
Comment (emilio) [inline tests/wpt/metadata/webgl/conformance-1.0.3/conformance/more/functions/bufferDataBadArgs.html.ini]: This should not be intermittent anymore after #10443.

File: tests/wpt/include.ini [hunk 2]
```diff
@@ -39,6 +39,8 @@ skip: true
         skip: false
 [cors]
   skip: false
+[webgl]
```
Comment (dzbarsky) [inline tests/wpt/include.ini]: o.o i'm confused, we're definitely running these tests already...

Comment (emilio) [inline tests/wpt/include.ini]: They were disabled again by @Ms2ger, since they caused a lot of intermittents :/

**Comment by notriddle** [comment]
That one failure was #10473

**Comment by jdm** [comment]
r? @Ms2ger 
Six try runs with no webgl failures.

**Comment by Ms2ger** [comment]
I'm confused by the commit message; you don't seem to disable any of the tests?

**Comment by jdm** [comment]
Originally there was one that needed disabling, but other PRs landed that removed that need.

**Comment by Ms2ger** [comment]
R+ with the commit message fixed

**Comment by emilio** [comment]
@bors-servo: try

(Test expectations might need to get updated after #10670)

**Comment by jdm** [comment]
```
  ▶ Unexpected subtest result in /webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:
  │ FAIL [expected PASS] WebGL test #24: contextA.getUniform(programS, locationSx) should be 3. Threw exception TypeError: contextA.getUniform is not a function
  │   → assert_true: contextA.getUniform(programS, locationSx) should be 3. Threw exception TypeError: contextA.getUniform is not a function expected true got false
  │ 
  │ reportTestResultsToHarness/<@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:87:9
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1397:20
  │ test@http://web-platform.test:8000/resources/testharness.js:495:9
  │ reportTestResultsToHarness@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:86:1
  │ testFailed@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:152:5
  │ shouldBe@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:229:9
  └ @http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:61:1

  ▶ Unexpected subtest result in /webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:
  │ FAIL [expected PASS] WebGL test #25: contextA.getUniform(programS, locationArray0) should be 123. Threw exception TypeError: contextA.getUniform is not a function
  │   → assert_true: contextA.getUniform(programS, locationArray0) should be 123. Threw exception TypeError: contextA.getUniform is not a function expected true got false
  │ 
  │ reportTestResultsToHarness/<@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:87:9
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1397:20
  │ test@http://web-platform.test:8000/resources/testharness.js:495:9
  │ reportTestResultsToHarness@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:86:1
  │ testFailed@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:152:5
  │ shouldBe@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:229:9
  └ @http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:62:1

  ▶ Unexpected subtest result in /webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:
  │ FAIL [expected PASS] WebGL test #20: getError expected: INVALID_OPERATION. Was NO_ERROR : after evaluating: contextA.uniform1i(locationSx, 3)
  │   → assert_true: getError expected: INVALID_OPERATION. Was NO_ERROR : after evaluating: contextA.uniform1i(locationSx, 3) expected true got false
  │ 
  │ reportTestResultsToHarness/<@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:87:9
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1397:20
  │ test@http://web-platform.test:8000/resources/testharness.js:495:9
  │ reportTestResultsToHarness@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:86:1
  │ testFailed@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:152:5
  │ glErrorShouldBe@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/resources/webgl-test-utils.js:1379:5
  │ shouldGenerateGLError@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/resources/webgl-test-utils.js:1355:5
  └ @http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:51:1

  ▶ Unexpected subtest result in /webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:
  │ FAIL [expected PASS] WebGL test #21: contextA.getUniform(programS, locationSx) threw exception TypeError: contextA.getUniform is not a function
  │   → assert_true: contextA.getUniform(programS, locationSx) threw exception TypeError: contextA.getUniform is not a function expected true got false
  │ 
  │ reportTestResultsToHarness/<@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:87:9
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1397:20
  │ test@http://web-platform.test:8000/resources/testharness.js:495:9
  │ reportTestResultsToHarness@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:86:1
  │ testFailed@http://web-platform.test:8000/webgl/conformance-1.0.3/resources/js-test-pre.js:152:5
  │ shouldGenerateGLError@http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/resources/webgl-test-utils.js:1353:5
  └ @http://web-platform.test:8000/webgl/conformance-1.0.3/conformance/uniforms/uniform-location.html:52:1
```

**Comment by jdm** [comment]
@bors-servo: retry
* #10846

**Pull request merged**